website/source/docs/agent/kv.html.md

```diff
@@ -14,7 +14,7 @@ The Consul KV datastore is located on the servers, but can be accessed by any ag
 
 ## Accessing the KV store
 
-The KV store can be accessed by the [consul kv CLI subcommands](/docs/commands/kv.html), [HTTP API](/api/kv.html), and Consul UI. To restrict access, enable and configure [ACLs](https://learn.hashicorp.com/consul/advanced/day-1-operations/acl-guide). Once the ACL system has been bootstrapped, users and services, will need a valid token with KV [privileges](/docs/agent/acl-rules.html#key-value-rules) to access the the data store, this includes even reads.  We recommend creating a token with limited privileges, for example, you could create a token with write privileges on one key for developers to update the value related to their application.
+The KV store can be accessed by the [consul kv CLI subcommands](/docs/commands/kv.html), [HTTP API](/api/kv.html), and Consul UI. To restrict access, enable and configure [ACLs](https://learn.hashicorp.com/consul/security-networking/production-acls). Once the ACL system has been bootstrapped, users and services, will need a valid token with KV [privileges](/docs/agent/acl-rules.html#key-value-rules) to access the the data store, this includes even reads.  We recommend creating a token with limited privileges, for example, you could create a token with write privileges on one key for developers to update the value related to their application.
```

Comment:
This could benefit from a word wrap while you're here so that future PRs are easier to skim.

Reply:
I updated the entire doc.